### PR TITLE
Update containerd task driver docs.

### PR DESCRIPTION
**Pull request by shishir-a412ed**

### CODE REVIEW

**Comment by shishir-a412ed** [comment]
ping @jrasell @tgross @notnoop

File: website/pages/docs/drivers/external/containerd.mdx [hunk 1]
```diff
@@ -234,6 +258,76 @@ before you can use `bridge` networks.
...
+
+```hcl
+ net.bridge.bridge-nf-call-arptables = 1
+ net.bridge.bridge-nf-call-ip6tables = 1
+ net.bridge.bridge-nf-call-iptables = 1
+```
+
+## Port Forwarding
+
+nomad supports both `static` and `dynamic` port mapping.
```
Comment (tgross) [inline website/pages/docs/drivers/external/containerd.mdx:283]:
```suggestion
Nomad supports both `static` and `dynamic` port mapping.
```

File: website/pages/docs/drivers/external/containerd.mdx [hunk 2]
```diff
@@ -234,6 +258,76 @@ before you can use `bridge` networks.
...
+  mode = "bridge"
+  port "http" {
+    to = 8080
+  }
+}
+```
+Here, nomad will allocate a dynamic port on the `host` and that port
+will be mapped to `8080` in the container.
+
+You can also read more about `network stanza` in the [`nomad official documentation`](https://www.nomadproject.io/docs/job-specification/network).
```
Comment (tgross) [inline website/pages/docs/drivers/external/containerd.mdx:317]: We don't need to put code blocks around `stanza`. Generally we don't want to inline links, but put them at the end of the page instead.

```suggestion
You can read more about configuring networking under the [`network`] stanza documentation.
```

File: website/pages/docs/drivers/external/containerd.mdx [hunk 3]
```diff
@@ -268,4 +362,6 @@ the external driver in the [plugin_dir][plugin_dir] directory.
 [plugin-options]: #plugin_options
 [host-network]: #host_network
 [`mount options`]: https://github.com/containerd/containerd/blob/9561d9389d3dd87ff6030bf1da4e705bbc024130/mount/mount_linux.go#L198-L222
+[`docker behavior`]: https://docs.docker.com/config/containers/container-networking/#dns-services
+[`allowed syscalls`]: https://github.com/containerd/containerd/blob/master/contrib/seccomp/seccomp_default.go#L51-L390
```
Comment (tgross) [inline website/pages/docs/drivers/external/containerd.mdx:366]:
```suggestion
[Moby repository]: https://github.com/moby/moby/blob/master/profiles/seccomp/default.json
[allowed syscalls]: https://github.com/containerd/containerd/blob/master/contrib/seccomp/seccomp_default.go#L51-L390
[`network`]: /docs/job-specification/network
[`service`]: /docs/job-specification/service
```

File: website/pages/docs/drivers/external/containerd.mdx [hunk 4]
```diff
@@ -234,6 +258,76 @@ before you can use `bridge` networks.
...
+You can also read more about `network stanza` in the [`nomad official documentation`](https://www.nomadproject.io/docs/job-specification/network).
+
+## Service discovery
+
+Nomad schedules workloads of various types across a cluster of generic hosts.
+Because of this, placement is not known in advance and you will need to use
+service discovery to connect tasks to other services deployed across your cluster.
+Nomad integrates with Consul to provide service discovery and monitoring.
+
+A [`service stanza`](https://www.nomadproject.io/docs/job-specification/service)
```
Comment (tgross) [inline website/pages/docs/drivers/external/containerd.mdx:326]: "stanza" doesn't need to be monospace, and usually we prefer the term "block" to "stanza".

```suggestion
A [`service`] block
```

File: website/pages/docs/drivers/external/containerd.mdx [hunk 5]
```diff
@@ -234,6 +258,76 @@ before you can use `bridge` networks.
...
+to allow container traffic through the bridge network to be routed via iptables.
+These tunables can be set as follows:
+
+```hcl
+ $ echo 1 > /proc/sys/net/bridge/bridge-nf-call-arptables
+ $ echo 1 > /proc/sys/net/bridge/bridge-nf-call-ip6tables
+ $ echo 1 > /proc/sys/net/bridge/bridge-nf-call-iptables
+```
+
+To preserve these settings on startup of a nomad client node, add a file
```
Comment (tgross) [inline website/pages/docs/drivers/external/containerd.mdx:271]:
```suggestion
To preserve these settings on startup of a Nomad client node, add a file
```

File: website/pages/docs/drivers/external/containerd.mdx [hunk 6]
```diff
@@ -123,6 +126,27 @@ config {
...
+  launched using `containerd-driver` will use host `/etc/resolv.conf`. This is
+  similar to [`docker behavior`][]. However, if you don't want to use
+  host DNS, you can turn off this flag by setting `host_dns=false`.
+
+- `seccomp` - (Optional) Enable default seccomp profile. List of [`allowed syscalls`][].
+
+- `seccomp_profile` - (Optional) Path to custom seccomp profile.
+  `seccomp` must be set to `true` in order to use `seccomp_profile`.
+
+  The default `docker` seccomp profile found [`here`](https://github.com/moby/moby/blob/master/profiles/seccomp/default.json)
```
Comment (tgross) [inline website/pages/docs/drivers/external/containerd.mdx:139]: Don't need the backticks here.

```suggestion
  The default `docker` seccomp profile found in the [Moby repository]
```

File: website/pages/docs/drivers/external/containerd.mdx [hunk 7]
```diff
@@ -268,4 +362,6 @@ the external driver in the [plugin_dir][plugin_dir] directory.
 [plugin-options]: #plugin_options
 [host-network]: #host_network
 [`mount options`]: https://github.com/containerd/containerd/blob/9561d9389d3dd87ff6030bf1da4e705bbc024130/mount/mount_linux.go#L198-L222
+[`docker behavior`]: https://docs.docker.com/config/containers/container-networking/#dns-services
```
Comment (tgross) [inline website/pages/docs/drivers/external/containerd.mdx:365]:
```suggestion
[Docker's behavior]: https://docs.docker.com/config/containers/container-networking/#dns-services
```

File: website/pages/docs/drivers/external/containerd.mdx [hunk 8]
```diff
@@ -123,6 +126,27 @@ config {
 }
 ```
 
+- `host_dns` - (Optional) `true` (default) or `false` By default, a container
+  launched using `containerd-driver` will use host `/etc/resolv.conf`. This is
+  similar to [`docker behavior`][]. However, if you don't want to use
```
Comment (tgross) [inline website/pages/docs/drivers/external/containerd.mdx:131]:
```suggestion
  similar to [Docker's behavior]. However, if you don't want to use
```

**Review by tgross — CHANGES_REQUESTED** [review]
Hi @shishir-a412ed and thanks for this PR!

File: website/pages/docs/drivers/external/containerd.mdx (hunk 1 above)
Comment (shishir-a412ed) [reply website/pages/docs/drivers/external/containerd.mdx]: Fixed.

File: website/pages/docs/drivers/external/containerd.mdx (hunk 2 above)
Comment (shishir-a412ed) [reply website/pages/docs/drivers/external/containerd.mdx]: Fixed.

File: website/pages/docs/drivers/external/containerd.mdx (hunk 3 above)
Comment (shishir-a412ed) [reply website/pages/docs/drivers/external/containerd.mdx]: Fixed.

File: website/pages/docs/drivers/external/containerd.mdx (hunk 4 above)
Comment (shishir-a412ed) [reply website/pages/docs/drivers/external/containerd.mdx]: Fixed.

File: website/pages/docs/drivers/external/containerd.mdx (hunk 5 above)
Comment (shishir-a412ed) [reply website/pages/docs/drivers/external/containerd.mdx]: Fixed.

File: website/pages/docs/drivers/external/containerd.mdx (hunk 6 above)
Comment (shishir-a412ed) [reply website/pages/docs/drivers/external/containerd.mdx]: Fixed.

File: website/pages/docs/drivers/external/containerd.mdx (hunk 7 above)
Comment (shishir-a412ed) [reply website/pages/docs/drivers/external/containerd.mdx]: Fixed.

File: website/pages/docs/drivers/external/containerd.mdx (hunk 8 above)
Comment (shishir-a412ed) [reply website/pages/docs/drivers/external/containerd.mdx]: Fixed.

**Comment by shishir-a412ed** [comment]
Hi @tgross, Thanks for the quick review! I have addressed your review comments. PTAL.

File: website/pages/docs/drivers/external/containerd.mdx [hunk 9]
```diff
@@ -123,6 +126,27 @@ config {
 }
 ```
 
+- `host_dns` - (Optional) `true` (default) or `false` By default, a container
+  launched using `containerd-driver` will use host `/etc/resolv.conf`. This is
+  similar to [Docker's behavior]. However, if you don't want to use
+  host DNS, you can turn off this flag by setting `host_dns=false`.
+
+- `seccomp` - (Optional) Enable default seccomp profile. List of [`allowed syscalls`][].
```
Comment (tgross) [inline website/pages/docs/drivers/external/containerd.mdx:134]: Sorry, looks like we missed a spot: https://nomad-i2tmn1oma.vercel.app/docs/drivers/external/containerd#seccomp

```suggestion
- `seccomp` - (Optional) Enable default seccomp profile. List of [allowed syscalls].
```

**Review by tgross — CHANGES_REQUESTED** [review]
Sorry, missed one more.

File: website/pages/docs/drivers/external/containerd.mdx (hunk 9 above)
Comment (shishir-a412ed) [reply website/pages/docs/drivers/external/containerd.mdx]: Fixed.

Comment (tgross) [reply website/pages/docs/drivers/external/containerd.mdx]: Thanks!

**Review by tgross — APPROVED** [review]
LGTM :+1:

**Pull request merged**